src/client/automation/playback/press/index.js

```diff
@@ -195,11 +169,42 @@ export default class PressAutomation {
             });
     }
 
-    _runCombination (keyCombination) {
-        if (this.proxylessInput)
-            return this._runCombinationViaCDP(keyCombination);
+    _calculateCDPEventSequence () {
+        const eventSequence = [];
+
+        for (const keyCombination of this.keyCombinations) {
+            const simulatedKeyInfos = getSimulatedKeyInfo(keyCombination);
+            let modifiersBit        = 0;
 
-        return this._runCombinationViaClientEventSimulation(keyCombination);
+            for (const keyInfo of simulatedKeyInfos) {
+                modifiersBit     |= getModifiersBit(keyInfo.key);
+                keyInfo.modifiers = modifiersBit;
+
+                changeLetterCaseIfNecessary(keyInfo);
+
+                eventSequence.push({
+                    type:    EventType.Keyboard,
```

Comment:
What about creating separate creators for events and send to them only one argument? For example: 
```js
createKeyboardEvent(options) {
  return {
    type: EventType.Keyboard,
    options
  }
}
```